### PR TITLE
[improve][doc] Add BackStage Docs theme to HTML and JavaDoc outputs. (SDKS-2334)

### PR DESCRIPTION
**Pull request by forgerock-chris**
# JIRA Ticket

[SDKS-2334](https://bugster.forgerock.org/jira/browse/SDKS-2334)

# Description

**Problem description:** 

[SDKS-2123](https://bugster.forgerock.org/jira/browse/SDKS-2123) provided the foundation of the kdoc kotlin docs generation (gradle tasks etc...). 
The generated docs however needs to be customized - i.e. use ForgeRock logo/favicon, footer, etc... 

**Acceptance criteria:** 

The generated kdocs uses ForgeRock branding (Logo, footer, navigation area)

**Notes:**

See https://kotlinlang.org/docs/dokka-html.html#configuration

# Definition of Done Checklist:

- [ ] Acceptance criteria is met.
- [ ] All tasks listed in the user story have been completed.
- [ ] Coded to standards.
- [ ] Ensure backward compatibility.
- [ ] API reference docs is updated.
- [ ] Unit tests are written.
- [ ] Integration tests are written.
- [ ] e2e tests are written.
- [ ] Functional spec is written/updated.
- [ ] contains example code snippets.
- [ ] Change log updated.
- [ ] Documentation story is created and tracked.
- [ ] Tech debts and remaining tasks are tracked in separated ticket(s).

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -8,10 +8,15 @@
...
-    ext.kotlin_version = '1.7.20'
+    ext {
+        kotlin_version = '1.7.20'
+        customCSSFile = projectDir.toString() + "/dokka/fr-backstage-styles.css"
+        customLogoFile = projectDir.toString() + "/dokka/logo-icon.svg"
+        customTemplatesFolder = file(projectDir.toString() + "/dokka/templates")
+    }
     repositories {
         google()
-        jcenter()
```
Comment (jeyanthanperiyasamy) [inline build.gradle:14]: Jcenter repositorties are deprecated , so using mavencentral is fine. But @spetrov  keep an eye this when you release ....

File: dokka/templates/base.ftl [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<#import "page_metadata.ftl" as page_metadata>
```
Comment (jeyanthanperiyasamy) [inline dokka/templates/base.ftl:1]: add copyright

File: dokka/templates/page_metadata.ftl [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<#macro display>
```
Comment (jeyanthanperiyasamy) [inline dokka/templates/page_metadata.ftl:1]: add copyright

**Review by jeyanthanperiyasamy — CHANGES_REQUESTED** [review]
add copyright ..everything else looks good for styling

File: dokka/templates/base.ftl (hunk 2 above)
Comment (forgerock-chris) [reply dokka/templates/base.ftl:3]: DOne.

File: dokka/fr-backstage-styles.css [hunk 4]
```diff
@@ -0,0 +1,146 @@
+:root {
```
[the remark this answers is not in the document]
Comment (forgerock-chris) [reply dokka/fr-backstage-styles.css:3]: Done.

**Review by spetrov — APPROVED** [review]
Looks great! 👍🏻

**Pull request merged**